OpenEdXMobile/src/main/java/org/edx/mobile/module/analytics/SegmentAnalytics.java

```diff
@@ -53,37 +53,40 @@ public SegmentAnalytics(@ApplicationContext @NonNull Context context, @NonNull C
         final int flushInterval = context.getResources().getInteger(R.integer.analytics_flush_interval);
 
```

Comment:
In my opinion, we can use this line at the top of the constructor instead:
```
if (!config.getSegmentConfig().isEnabled()) return;   
```
What do you think?

Reply:
That's 63 changes better